espnet2/asr/espnet_model.py

```diff
@@ -93,20 +99,39 @@ def __init__(
             self.ctc = None
         else:
             self.ctc = ctc
-        self.rnnt_decoder = rnnt_decoder
+
+        self.transducer_decoder = transducer_decoder
+        self.joint_network = joint_network
+
+        self.criterion_transducer = TransLoss("warp-transducer", self.blank)
```

Comment:
Hi, I think we currently do not have a default installation of warp transducer loss, probably we can add a condition here of whether to initial transducer loss here?